src/lime/graphics/Renderer.hx

```diff
@@ -30,15 +30,17 @@ class Renderer {
 	private function createGLContext(failIfMajorPerformanceCaveat:Bool):Void {
 		var transparentBackground = Reflect.hasField(window.config, "background") && window.config.background == null;
 		var colorDepth = Reflect.hasField(window.config, "colorDepth") ? window.config.colorDepth : 16;
+		var alpha = Reflect.hasField(window.config, "alpha") ? window.config.alpha : false;
 
 		var options = {
-			alpha: (transparentBackground || colorDepth > 16) ? true : false,
+			alpha: (alpha || transparentBackground || colorDepth > 16) ? true : false,
```

Comment:
This whole construiction is kinda flimsy. What if we explicitly specify `alpha`  to `false`, and then it would get passed as `true` due to the other conditions in this branch. That sounds confusing...

Not that the code here is nice anyway, so it's not a strong requirement, but handling some invariants would be helpful I guess, maybe add a note or TODO regarding this.

Reply:
I'm not sure, depends on how useful these background/colorDepth arguments are and if we're using them (i know we set at least background. I would love to clean it up these options at some point, but that can be initiated with explicitly passing `alpha` as you suggest, indeed. After all it's our fork and we can do whatever we feel nicer :)